README.md

```diff
@@ -1 +1,42 @@
-# Code Reviews
\ No newline at end of file
+# Code Reviews
+
+## Overview
+
+This repository is designed to teach students the code review process.  There are several pull requests, PRs, with sample code submissions.  Students will practice reviewing these submissions and then discuss their comments for clarity, etiquette and correctness.
+
+## How This Will Work
+
+### Reviewing PRs
+
+Each student will open 3 PRs in separate tabs in their browser.  The PRs are assigned by table.  They will review the code adding inline comments, questions and suggested changes.  They will **not** submit the review.  Instead, student will wait until all students are finished reviewing the PR.  
+
+Then students will submit their reviews and decide if they accept the PR, or want to request changes.
+
+Students are asked to look for:
+
+* Do I understand what this code is doing? (Readability)
+* Are there things wrong with this code? (Correctness)
+* Are there changes (Improvements) I can suggest to make the code more:
+  * Readable
+  * Efficient
```

Comment:
I worry about making the students think about efficiency this early.  Especially since the students haven't learned big O notation/algorithmic efficiency yet.

Reply:
This is for students in internship.  They damm well better have learned.  :p